packages/katello/katello/hostname-change.rb

```diff
@@ -598,7 +598,7 @@ def default_installer
       # always disable system checks to avoid unnecessary errors. The installer should have
       # already ran since this is to be run on an existing system and installer checks would
       # have already been skipped
-      "#{@options[:program]}-installer --scenario #{@options[:scenario]} -v --disable-system-checks"
+      "#{@options[:program]}-installer --scenario #{@options[:scenario]} -v"
```

Comment:
https://github.com/theforeman/foreman-packaging/pull/8767

I opted for keeping the scenario, as I recall there is some weirdness around that in the code and I wanted to avoid any surprises for now.